pasion_preciouskaye/index.html

```diff
@@ -0,0 +1,186 @@
+    <title>Pasion,Preciouskaye Page</title>
+    <link rel="stylesheet" href="style.css" />
+    <meta
+      name="viewport"
+      content="width=device-width, 
+    initial-scale=1.0"
+    />
+  </head>
+  <body>
+    <div class="position">
+    <div class="name">
+      <h1>Pasion Precious Kaye</h1>
+    </div>
+
+    <div class="hobbies">
+      <h2>My interests and hobbies</h2>
+      <ul>
+        <li>Reading Books</li>
+        <li>Listening Music</li>
+        <li>Sleeping</li>
+        <li>Playing Mobile Games</li>
+        <li>Cooking</li>
+      </ul>
+    </div>
+
+    <div class="works">
+      <h2>My Works</h2>
+      <ol>
+        <li><a href="example1.html">Example 1</a></li>
+        <li><a href="example2.html">Example 2</a></li>
+        <li><a href="example3.html">Example 3</a></li>
+        <li><a href="example4.html">Example 4</a></li>
+        <li><a href="example5.html">Example 5</a></li>
+      </ol>
+    </div>
+
+    <div class="comment">
+      <h2>Testimonials</h2>
+      <ul>
+        <li>
+          Her innovative ideas and flawless execution make her a great team
+          member. Keep up the great work!" - Alona
+        </li>
+        <li>
+          Her work shows her creativity. It also shows how well she used her
+          imagination and adapt it into her work. - Sofia
+        </li>
+        <li>You did a great webpage. Good job! - Eliza</li>
+        <li>
+          The user-friendly interface of the website makes it easy for new
+          visitors to navigate. - Mikaella
+        </li>
+      </ul>
+    </div>
+  </div>
+    
+  <div class="goals">
+    <h2>My Goals in Life</h2>
+    <ul>
+      <li>Finish My Study</li>
+      <li>Get my Dream Job</li>
+      <li>Traveling with My Love ones</li>
+      <li>Buy our Own House</li>
+      <li>Make My Mother</li>
+    </ul>
+  </div>
+
+  <div class="yourcomment">
+    <h2> Your Comment About My Goals</h2>
+    <ul>
+      <li>Hope you'll achieve it all - Sofia</li>
+      <li>Great Job! - Eliza</li>
+      <li>You've got this! :D - Mikaella</li>
+    </ul>
+  </div>
+   
+    <div class="images">
+      <h2>Gallery</h2>
+      <div class="image-item">
+      <img
+        src="https://w0.peakpx.com/wallpaper/430/970/HD-wallpaper-doraemon-art-cute.jpg"
+      />
+      <img
+        src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcQyAv_4V8XzSLF2lTUeMpXS7p0nH-8QomfB5zzxh9INxw&s"
+      />
+      <img
+        src="https://i.pinimg.com/originals/05/f6/05/05f605183b35d08386e122bf5a767d63.jpg"
+      />
+      <img
+        src="https://yt3.googleusercontent.com/HskYwAGlO-VcPpnUNJcfTKBbFrj2LNrRbrm1Ht5cuHhc621f2Tgzf-deKf3uL0qMY8_KRApMoZg=s900-c-k-c0x00ffffff-no-rj"
+      />
+      <img
+        src="https://i.pinimg.com/236x/b7/91/a9/b791a92b8fc7819738019516d168ed25.jpg"
+      />
+      <img
+      src="https://i.pinimg.com/736x/69/5f/2f/695f2f95029f73e501cf28263b5aa492.jpg"
+    />
```

Comment:
https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L87

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L90

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L93

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L96

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L99

https://github.com/PUP-BSIT/exercise-13-tech_nx/blob/c0e8cf05d5790a8696f6421541e2f159ccaffc67/pasion_preciouskaye/index.html#L102


You may just attach the </ at the end of the line so it won't have an unnecessary excess line of code.

https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit?fbclid=IwZXh0bgNhZW0CMTAAAR04rIg_SBkft7ED5muFsxm9FUkijQq-aweBdgT21-tlg-UpXKtfkO4Nb9s_aem_AanCGQNw3v8o-ZJK1bprYBSLNcABosgmI-0dnlGiU_GSgrlxM4drKiB_S99jcEItxjMmHfHXZ64dF_L3NbTavA6s#slide=id.g24da8de92d7_0_0